### DISCUSSION TITLE
On Android: new created list not shown in selection.

### DISCUSSION THREAD

**Discussion by M2vH** [community/community#18851] [Other Feature Feedback, Questions, & Ideas]
If I want to add repo to a list, and I create the list inside the add dialog, the new created list is not shown in the selection list.

**Comment by darkvertex** [comment]
> If I want to add repo to a list, and I create the list inside the add dialog, the new created list is not shown in the selection list. 

For me not only did this happen but when I popped the lists list again, the new list was the only list there! Did I just lose all my lists?! 🤔

**Reply by SharakPL** [reply]
The same for me. It should definitely show all the lists and not just the one created... and of course show the list right after it is created.

**Comment by pratikkabade** [comment]
I can see it. 
There might be issue on your device.
Try updating GitHub or reinstalling it.

**Reply by SharakPL** [reply]
I already have the latest version available installed. Cache cleared. The result is still the same.

**Reply by pratikkabade** [reply]
Which device are you using?

**Reply by SharakPL** [reply]
![Screenshot_20220630-190239-180](https://user-images.githubusercontent.com/5007456/176736080-e9561288-3927-4c3b-b58f-a3c5a5d2ca6b.png)
on Android 11

**Reply by SharakPL** [reply]
Just updated to 1.71.1 (666) and it's the same. Still only the one list I created show up. Also with this update I can't edit my messages here.

At least my previous message I couldn't. Maybe because it was written in previous version.

**Reply by SharakPL** [reply]
Nope, it's when I restart GitHub app I'm losing the ability to edit my messages. I can only edit right away after message is created.

**Reply by pratikkabade** [reply]
Yes we cant edit message after i guess 24 hours.
And I still think it must be device bug, did you tried reinstalling GitHub?

**Reply by SharakPL** [reply]
Let me be clear on that. First time I wanted to add something to the list, GH app had some suggestions for the name. I liked that but that time I wanted different name. But since creating this first list those suggestions have never been shown again on other repos.

**Reply by SharakPL** [reply]
And no, it's not 24 hours for post edit. I could edit above post right after creating it, but few seconds later after app's restart the Edit option is gone 😞

I can only edit through desktop browser, not through mobile app.

**Comment by Patrick2239** [comment]
Do more

**Comment by skzwow** [comment]
https://www.planeo.sk/katalog/1323177-carneo-gear-deluxe-gold-smart-hodinky.html#
